includes/Integrations/AMP.php

```diff
@@ -388,9 +388,16 @@ protected function get_validated_url_post_type( $post_id ) {
 			return null;
 		}
 
-		$queried_object = get_post_meta( $post->ID, '_amp_queried_object', true );
+		$queried_object = (array) get_post_meta( $post->ID, '_amp_queried_object', true );
```

Comment:
I thought were trying to remove the castings?

Reply:
Maybe some leftover, I'll have to check.